Our packaging manifest relies on "35" in pathnames and dependency package
names. Probably that can be played around by macros at a later point, but
for now it is easier to bolt the version.

--- gcm-1.1.0-orig/gnome_connection_manager.py	2012-07-05 16:18:58.000000000 +0200
+++ gcm-1.1.0/gnome_connection_manager.py	2012-07-05 23:59:58.000000000 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python3.5
 # -*- coding: UTF-8 -*-
 
 # Python module gnome_connection_manager.py
